core/controllers/base.py

```diff
@@ -216,14 +218,8 @@ def dispatch(self):
             return
 
         if self.user_is_scheduled_for_deletion:
-            # In DEV_MODE, clearing cookies does not log out the user, so we
-            # force-clear them by redirecting to the logout URL.
-            if constants.DEV_MODE:
-                self.redirect(
-                    current_user_services.create_logout_url(
```

Comment:
Is there a reason we are dropping the use of current_user_services.create_logout_url()?

Reply:
Yes, how `current_user_services.create_logout_url()` works is that it logs the user out not only from Oppia but also from all their Google Accounts. 

We used it on local where the Google Accounts are mocked so it is not a problem, but I have replaced it on local too with the removal of the cookie that is used to identify user on local servers (see changes in `_clear_login_cookies`). This way we close that gap between local dev/prod and production a little bit again.

Reply:
This sounds good and I like how you're handling it. Thanks for the explanation!